anvil/src/input_handler.rs

```diff
@@ -82,10 +83,10 @@ impl<BackendData: Backend> AnvilState<BackendData> {
                     )
                     .spawn()
                 {
-                    error!(self.log,
-                        "Failed to start program";
-                        "cmd" => cmd,
-                        "err" => format!("{:?}", e)
+                    error!(
+                        cmd,
+                        err = ?e,
```

Comment:
io::Error implements Display, so I'm not sure why we use the Debug representation here